simulator/CARLA_Autoware/carla_pointcloud_interface/src/carla_pointcloud_interface/carla_pointcloud_interface_node.cpp

```diff
@@ -0,0 +1,53 @@
+// Copyright 2024 Raditya.
+
+#include "carla_pointcloud_interface/carla_pointcloud_interface_node.hpp"
+
+#include <pcl_ros/transforms.hpp>
+
+#include <pcl_conversions/pcl_conversions.h>
+#include <yaml-cpp/yaml.h>
```

Comment:
I think we can remove this include and all dependencies on `YAML_CPP`.

Reply:
> #include <pcl_conversions/pcl_conversions.h>
> #include <yaml-cpp/yaml.h>
Removed above dependencies.

> #include <pcl_ros/transforms.hpp>
Keeping above, since it is still needed to transform the frame_id on line 25.